{"name": "test_case3_login_user_with_incorrect_credentials[chromium]", "status": "passed", "description": "Should test login user with incorrect credentials", "attachments": [{"name": "case3:login_user_with_incorrect_credentials", "source": "ae357724-2085-497a-909a-3b10ba872dcd-attachment.png", "type": "image/png"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1677581769774, "stop": 1677581772946, "uuid": "3a0fadcf-52b2-4fb4-9a31-ce32e2e10760", "historyId": "aff4e8061918338cd68af55074c712fb", "testCaseId": "ac14b3651118b74ba9c265dfa61da173", "fullName": "tests.test_login.TestAutomationExerciseLogin#test_case3_login_user_with_incorrect_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestAutomationExerciseLogin"}, {"name": "host", "value": "DESKTOP-9ICER92"}, {"name": "thread", "value": "14372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}